mentor_classifier/tests/helpers.py

```diff
@@ -5,6 +5,110 @@
 # The full terms of this copyright and license should always be found in the root directory of this software deliverable as "license.txt" and if these terms are not found with this software, please contact the USC Stevens Center for the full license.
 #
 from os import path
+from dataclasses import dataclass, asdict, field
+from typing import List, Optional
+
+
+@dataclass
+class MentorQuestion:
+    question_id: str
+    question: str
+    paraphrases: List[str]
+    answer: str
+    topic: str
+
+
+@dataclass
+class Subject:
+    name: str
+
+
+@dataclass
+class Topic:
+    name: str
+
+
+@dataclass
+class Question:
+    _id: str
+    question: Optional[str] = None
+    type: Optional[str] = None
```

Comment:
in python, you almost never want to use `Optional[str]`, instead use empty string, e.g.

```python
@dataclass
class Question:
    _id: str
    question = ""
```

...remember that python has an expansive view of `falsy`ness...so

```python
bool("")  # False
bool([])  # False
bool({})  # False
```


question